.github/workflows/bump-internal-repo.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@master
         with:
           repository: grouparoo/internal
-          token: ${{ secrets.GITHUB_ACCESS_TOKEN }}
```

Comment:
it now seems that secrets in github actions cannot contain `github` (the string)